src/utils/figmaTheme/templateResolvers/parsers/boxShadowParser.js

```diff
@@ -4,15 +4,19 @@ export const boxShadowParser = (value, root) => {
   if (typeof value.value === 'string') {
     return replaceTemplateValue(value.value, root);
   }
-  const mappedStrings = value.value.map(
-    ({ blur, color, spread, type, x, y }) =>
-      `${replaceTemplateValue(x, root)}px ${replaceTemplateValue(
-        y,
-        root
-      )}px ${replaceTemplateValue(blur, root)}px ${replaceTemplateValue(
-        spread,
-        root
-      )}px ${replaceTemplateValue(color, root)}`
-  );
-  return `${mappedStrings.join(', ')}`;
+
+  const buildShadow = ({ blur, color, spread, type, x, y }) =>
```

Comment:
What is `type` used for in this?

Reply:
`type` is set as 'DropShadow' but I may believe it can be set as `inset`. It's unused so I can remove it for now. I'll update an eslint rule later that makes sure we have `no-unused`